src/alpaca_eval/models_configs/Infinity-Instruct-7M-Gen-Llama3_1-8B/configs.yaml

```diff
@@ -0,0 +1,13 @@
+Infinity-Instruct-7M-Gen-Llama3_1-8B: # this should be the same as the name as the current directory
```

Comment:
why are those new files? shouldn't they be updated ones? 
I.e. can you remove the old ones

Reply:
I'm sorry about that.